src/democracys-library.ts

```diff
@@ -1,33 +1,333 @@
+import { headerImageUrl } from './data/header-image';
+import { didYouKnow, Factoid } from './data/did-you-know';
+import './ti-tle';
 
 @customElement('ia-democracys-library')
 export class IaDemocracysLibrary extends LitElement {
-  @property({ type: String }) title = 'Hey there';
+  @property({ type: Array }) carousel1: CarouselCard[] = carousel1;
 
-  @property({ type: Number }) counter = 5;
+  @property({ type: Array }) carousel2: CarouselCard[] = carousel2;
 
-  private increment() {
-    this.counter += 1;
+  @property({ type: Array }) didYouKnow: Factoid[] = didYouKnow;
+
+  @property({ type: Array }) highlights: card[] = collectionsToHighlight;
+
+  @property({ type: String }) fontSource = '';
+
+  firstUpdated(): void {
+    const fontStyle = document.createElement('style');
+    fontStyle.innerHTML = `
+    @font-face {
+      font-family: "Teko";
+      src: url(https://archive.org/cors/democracys-library/web-component/Teko-SemiBold.ttf);
+    }`;
+    fontStyle.setAttribute('id', 'teko-font');
+    document.head.appendChild(fontStyle);
+  }
+
+  disconnectedCallback(): void {
+    const fontLink = document.head.querySelector('#teko-font');
+    if (fontLink) {
+      document.head.removeChild(fontLink);
+    }
+  }
+
+  // <i><img src="https://archive.org/download/democracys-library/web-component/help.svg" alt="help icon"></i>
+  get factoids(): TemplateResult[] {
+    return this.didYouKnow.map((fact: Factoid, i) => {
+      const tintColor = i % 2 === 0 ? 'green' : 'yellow';
+      return html`
+        <article>
+          <div class="title">
+            <ti-tle class=${tintColor}><span>DID YOU KNOW?</span></ti-tle>
+          </div>
+          <p class="full-width">${fact.details}</p>
+        </article>
+      `;
+    });
+  }
+
+  resourceCard(
+    card: card,
+    tintColor: 'yellow' | 'green' | undefined
+  ): TemplateResult {
+    return html`
+      <article>
+        <div class="title">
+          <ti-tle class=${tintColor ?? ''}>${card.title}</ti-tle>
+        </div>
+        <item-preview-image
+          src=${card.image}
+          alt="preview image."
+          class=${tintColor ?? ''}
+        ></item-preview-image>
+        <p>${card.blurb}</p>
+        <a href=${card.link} tab="_blank"
+          >Browse the ${card.title} collection</a
+        >
+      </article>
+    `;
+  }
+
+  carouselCard(
+    card: CarouselCard,
+    tintColor: 'yellow' | 'green' | undefined
+  ): TemplateResult {
+    const url = `https://archive.org/details/${card.id}`;
+    return html`
+      <a href=${url}>
+        <item-preview-image
+          src=${card.image}
+          alt=${`Go to: ${card.title}`}
+          class=${tintColor ?? ''}
+        ></item-preview-image>
+      </a>
+    `;
+  }
+
+  get topCarousel(): TemplateResult {
+    return html`
+      <ti-tle class="green">${carousel1Title}</ti-tle>
+      <section id="carousel-1" class="carousel">
+        ${this.carousel1.map((card, i) => {
```

Comment:
we can refactor this to use repeat directive, https://lit.dev/docs/api/directives/#repeat